docs/ci-cd-integration/deploy-from-github.md

```diff
@@ -159,7 +172,7 @@ jobs:
       with:
             registry: ghcr.io
             username: ${{ github.repository_owner }}
-            password: ${{ secrets.GH_TOKEN }}
+            password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
There are other references of "GH_TOKEN" in this page, we should keep it consistent.

Reply:
Right, changed it 👍